### DISCUSSION TITLE
Does Objeck support Exceptions?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#352] [General]
I have read `Getting Started` and it doesn't mention anything about Exceptions.

**Comment by objeck** [comment]
The language does not support exceptions by design. As a former Java developer, I noticed that many programmers caught and logged exceptions and threw them up the stack where they were logged or ignored. I also saw that programmers would catch "catch-all" generic runtime exceptions. I like the C paradigm of checking the return values from functions.
